client/source/js/modules/admin/manage-projects-ctrl.js

```diff
@@ -0,0 +1,125 @@
+define(['./module', 'angular', 'underscore'], function (module, angular, _) {
+  'use strict';
+
+  module.controller('AdminManageProjectsController', function ($scope, $http, info, projects, users, activeProject, UserManager, modalService) {
+    $scope.users = users.data.users;
+    $scope.users = _.compact(_.map(_(projects.data.projects).groupBy(function (p) {return p.user_id;}), function (projects, userId) {
+      var user = _.findWhere($scope.users, {id: parseInt(userId)});
+      return user.email===UserManager.data.email ? undefined :{
+        projects: projects,
+        data: user
+      };
+    }));
+
+    /**
+     * Regenerates workbook for the given project `name`
+     * Alerts the user if it cannot do it.
+     *
+     */
+    $scope.workbook = function (name, id) {
+      // read that this is the universal method which should work everywhere in
+      // http://stackoverflow.com/questions/24080018/download-file-from-a-webapi-method-using-angularjs
+      window.open('/api/project/workbook/' + id, '_blank', '');
+    };
+
+    /**
+     * Opens to edit an existing project using `name` in /project/create screen
+     *
+     * Alerts the user if it cannot do it.
+     */
+    $scope.edit = function (name, id) {
+      $http.get('/api/project/open/' + id)
+        .success(function (response) {
+          if (response && response.status === 'NOK') {
```

Comment:
@kitze @Ashalynd Every response with status code 'NOK' will have HTTP status code 500 never reach the success callback. https://github.com/StarterSquad/Optima/blob/develop/server/src/api.py#L47

Yesterday a PR was merged showing a modal.inform whenever a HTTP request made from Angular fails. Because if this error handling here is not necessary anymore.